crates/ruff_server/docs/setup/HELIX.md

```diff
@@ -0,0 +1,30 @@
+## Helix Setup Guide for `ruff server`
+
+First, open the language configuration file for Helix. On Linux and macOS, this will be at `~/.config/helix/languages.toml`,
+and on Windows this will be in `%AppData%\helix\languages.toml`.
+
+Add the language server by adding:
+
+```toml
+[language-server.ruff]
+command = "ruff"
+args = ["server", "--preview"]
```

Comment:
Nit: formatting here is different (no space after and before hard brackets) unlike the below. I'd probably just remove the spacing in both cases.